File: backend/user-service/src/main/java/com/yinling/user/dto/UserLoginResponse.java
package com.yinling.user.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * 用户登录响应DTO
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class UserLoginResponse {

    private Long userId;

    private String phone;

    private String nickname;

    private String avatar;

    private Integer userType;

    private String accessToken;

    private String refreshToken;

    private Long expiresIn;  // Token过期时间（秒）
}

File: android-elderly-app/app/src/main/java/com/yinling/elderly/data/model/UserLoginResponse.kt
package com.yinling.elderly.data.model

/**
 * 用户登录响应
 */
data class UserLoginResponse(
    val code: Int,
    val message: String,
    val data: LoginData?
)

data class LoginData(
    val userId: Long,
    val phone: String,
    val nickname: String,
    val avatar: String?,
    val userType: Int,
    val accessToken: String,
    val refreshToken: String,
    val expiresIn: Long
)
